client/js/screens/Ethics/Ethics.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {Text} from 'react-native';
+// import CText from '../../components/CustomText';
```

Comment:
same as above.

Reply:
Part of figuring out custom text, will be needed later